diff --git a/src/styles/animations.styl b/src/styles/animations.styl
index 2e8e285..b6758b5 100644
--- a/src/styles/animations.styl
+++ b/src/styles/animations.styl
@@ -188,7 +188,8 @@ $animationTextColor= #dcbc94;
   #text24 { animation: animationNormalFade $TIME_FACTOR * 2s linear $TIME_FACTOR * 72.5s - $start_at_with_factor; }
   #text25 { animation: animationNormalFade $TIME_FACTOR * 2s linear $TIME_FACTOR * 79.7s - $start_at_with_factor; }

-  #text26 { animation: logoAnimation $TIME_FACTOR * 7s linear $TIME_FACTOR * 86.75s - $start_at_with_factor; }
+  // #text26 { animation: logoAnimation $TIME_FACTOR * 7s linear $TIME_FACTOR * 86.75s - $start_at_with_factor; }
+  #text26 { animation: logoAnimation $TIME_FACTOR * 7.15s linear $TIME_FACTOR * 86.6s - $start_at_with_factor; }

   #text27 { animation: animationNormalFade $TIME_FACTOR * 2.056s linear $TIME_FACTOR * 94s - $start_at_with_factor; }
   #text28 { animation: animationNormalFade $TIME_FACTOR * 2.056s linear $TIME_FACTOR * 96.2s - $start_at_with_factor; }
